sys/isrpipe/Kconfig

```
@@ -15,4 +15,4 @@ menuconfig MODULE_ISRPIPE
 config MODULE_ISRPIPE_READ_TIMEOUT
     bool "ISR Pipe read with timeout"
     depends on MODULE_ISRPIPE
-    select MODULE_XTIMER
+    select ZTIMER_USEC
```

Should is rather be:
```suggestion
    select MODULE_ZTIMER_USEC
```

? or maybe there's a subtlety that I miss :)

---

I know, wild and wacky.  Here is the PR that introduced it: #17165

It is to prevent some circular dependency things... It would take me some time to get into it (I always have to retrain myself) but I can try to document it a bit better if the PR or comment for ZTIMER_USEC is not clear.